_posts/2016-05-11-android-testing-with-kotlin.md

```diff
@@ -0,0 +1,421 @@
+---
+layout: post
+title:  "Using Kotlin to test Android applications"
+date:   2016-05-11 00:00:00
+categories: Greenhouse update
```

Comment:
More of a note for us for the future:
We don't currently make use of the categories at all, it seems. If we plan to start doing so in the future, though, we should define a few of these and use them as appropriate. i.e. everything should not be an "update"